src/random_test.cpp

```diff
@@ -2679,11 +2679,16 @@ void Table::Compare_between_engine(const std::string &sql, Thd1 *thd) {
   set_default();
 }
 
+  // Data structures for recent queries
+  static std::deque<std::string> recent_queries;
```

Comment:
Is it used any where?

Reply:
I have corrected the logic and removed it